File: An2/MAP/MAP-Java/MAP-Lab6(NB)/src/Model/Student.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package Model;

/**
 *
 * @author mihai
 */
public class Student implements Comparable<Student> {
    private String name;
    private int id;
    private int grade;

    
    public Student (int id, String name, int grade) {
        this.name = name;
        this.id = id;
        this.grade = grade;
    }
    
    public float average(){
        return grade;
    }
    
    public int getID() {
        return this.id;
    }
    
    public int getGrade() {
        return this.grade;
    }
    
    public String getName(){
        return this.name;
    }
    
    @Override
    public String toString() {
        return id + " " + name + " " + grade;
    }

    @Override
    public boolean isGreaterThan(Student student) {
        return grade > student.grade;
    }
}

File: An2/Sem1/MAP/MAP-C#/MAP-Lab5/MAP-Lab5/Domain/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MAP_Lab5.Domain
{
    class Student
    {
        private String name;
        private int id;
        private int grade;

        Student()
        {
            name = null;
            id = 0;
            grade = 0;
        }

        public Student(String name, int id, int grade)
        {
            this.name = name;
            this.id = id;
            this.grade = grade;
        }

        public int getId()
        {
            return this.id;
        }

        public int getGrade()
        {
            return this.grade;
        }

        public string getName()
        {
            return this.name;
        }

        public String toString()
        {
            return name + " " + id + " " + grade;
        }
    }
}
